WordPress/gutenberg
# Remove role attributes on SVGs meant for "decoration" to improve a11y

## Description
Fixes #38286 

In many places throughout Gutenberg, we render SVGs with `role="img"` but we do not include an `aria-label`. At first glance this may seem ok because in all instances we also have something like this:

```
<svg role="img" aria-hidden="true" focusable="false" width="24" height="24" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg">
      ...
</svg>
```

However, `aria-hidden="true" focusable="false"` is sufficient here. Since we are including `role="img"`, there should also be an `aria-label` as described here in the [MDN Wed Docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role#svg_and_roleimg). The solution is to simply remove the `role` attribute. Refer to #38286 for further discussion on why this is the correct approach. 

## Testing Instructions

1. Using the latest version of Gutenberg, add a Social Icons block to the page.
2. Add a couple social links
3. Preview the page on the frontend and notice `role="img"` is no longer added to all SVG icons. 🙌

## Screenshots 

## Types of changes
Bug fix

## Checklist:
